src/js/App/Header/Tools.js

```diff
@@ -1,12 +1,11 @@
 import React, { Component } from 'react';
-import { Button } from '@patternfly/react-core/dist/esm/components/Button';
-import { Toolbar, ToolbarGroup, ToolbarItem } from '@patternfly/react-core/dist/esm/layouts/Toolbar';
-import { Tooltip } from '@patternfly/react-core/dist/esm/components/Tooltip';
-import { DropdownItem } from '@patternfly/react-core/dist/esm/components/Dropdown';
-import QuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/question-circle-icon';
-// import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
-// import BellIcon from '@patternfly/react-icons/dist/esm/icons/bell-icon';
-import CogIcon from '@patternfly/react-icons/dist/esm/icons/cog-icon';
+import { Button } from '@patternfly/react-core';
+import { Toolbar, ToolbarGroup, ToolbarItem } from '@patternfly/react-core';
+// eslint-disable-next-line no-unused-vars
+import { Tooltip } from '@patternfly/react-core';
```

Comment:
You can remove this component, since it's not used anywhere.

Reply:
done